cEP-0004.md

```diff
@@ -95,5 +113,14 @@ of them.
 Usage of letters would make usage of coala faster, as one could just remember
 that `a` stands for 'apply patch' for instance, and they would not have to check
 the list for every result to see what number 'apply patch' has been assigned.
-This change would also drastically reduce the required minimum amount of
-characters needed for th
+
+
+### Allow chaining actions in a single input
```

Comment:
We have coafile:

[sec1]
bears = SpaceConsistencyBear, PyUnusedCodeBear, QuotesBear
files = *.rst
language = Python
preferred_quotation = '

[sec2]
files = LICENSE, ISSUE_TEMPLATE, PULL_REQUEST_TEMPLATE.md, MANIFEST.in
bears = SpaceConsistencyBear

We will generate a patch with SpaceConsistencyBear that will apply all changes for both files from [sec1] and [sec2]. But before this we could give user a preview of what changes will be apply like:

SpaceConsistencyBear in LICENSE
<changes to be made>

SpaceConsistencyBear in README.rst:
<changes to be made>

and so on.